libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -451,6 +451,33 @@ void ofFbo::clear() {
 #endif
 }
 
+
+//--------------------------------------------------------------
+void ofFbo::clearColorBuffer(const ofFloatColor & color){
+	glClearBufferfv(GL_COLOR, 0, &color.r);
+}
+
+
+//--------------------------------------------------------------
+void ofFbo::clearColorBuffer(size_t buffer_idx, const ofFloatColor & color){
+	glClearBufferfv(GL_COLOR, buffer_idx, &color.r);
+}
+
+//--------------------------------------------------------------
+void ofFbo::clearDepthBuffer(float value){
+	glClearBufferfv(GL_DEPTH, 0, &value);
+}
+
+//--------------------------------------------------------------
+void ofFbo::clearStencilBuffer(float value){
+	glClearBufferfv(GL_STENCIL, 0, &value);
+}
+
+//--------------------------------------------------------------
+void ofFbo::clearDepthStencilBuffer(float depth, float stencil){
```

Comment:
i believe stencil shoud be an int value (see: https://www.opengl.org/sdk/docs/man/html/glClearBuffer.xhtml)